### DISCUSSION TITLE
Add Page specific Keywords

### DISCUSSION THREAD

**Discussion by Jh123x** [jpanther/congo#410] [Ideas]
A keywords section for each page.

This will improve per-page SEO for topics defined on the page.
For example, if one of my pages is about cooking, I can add a cooking keyword to allow search engines to relate my page to a cooking page.

## Proposed Syntax

Maybe a syntax that looks like this can allow users to add many keywords without cluttering up the page.

```
---
title: My Blog
description: ......
keywords:
  - "blog"
  - "my blog"
---

.....

```

This can be combined with the current partial in header using the snippet below.

```html
  {{ with union .Site.Params.keywords .Params.keywords -}}
    <meta name="keywords" content="{{ delimit . `, `}}" />
  {{- end }}
```

**Comment by jpanther** [comment]
Thanks for the suggestion. This feature will be included in the next theme release.
